packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -395,6 +516,15 @@ export const toStyles = ( tree, blockSelectors, hasBlockGapSupport ) => {
 				`${ duotoneSelector }{${ duotoneDeclarations.join( ';' ) };}`;
 		}
 
+		// Process blockGap and layout styles.
+		ruleset += getLayoutStyles(
```

Comment:
I just realised we're not checking here whether or not the block opts-in to the layout support, so this will fire on each block type. We should add a check here, and in the PHP implementation. I'll follow up tomorrow!